courses/machine_learning/deepdive2/image_classification/labs/1_mnist_linear.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<p>Keras'es -&gt; Keras'</p><p>Reminded me of Gollum with Baginses haha</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>